### PR TITLE
REFS maintenance - Changes some broker registration langauge for maine.

### PR DESCRIPTION
**Pull request by ZASMan**
### Master Pivotal ticket
*https://www.pivotaltracker.com/n/projects/2481183/stories/179688312

### CODE REVIEW

File: app/views/ui-components/v1/forms/broker_registration/_broker_agency_information.html.slim [hunk 1]
```diff
@@ -1,4 +1,4 @@
-h4.mb-2 NOTE: If you are not associated with a broker agency, please enter your personal information. Do not enter a General Agency or Third Party Administrator‘s Information.
+h4.mb-2 NOTE: If you are not associated with a #{l10n("broker_agency")}, please enter your personal information. Do not enter a General Agency or Third Party Administrator‘s Information.
```
Comment (bill-transue) [inline app/views/ui-components/v1/forms/broker_registration/_broker_agency_information.html.slim:1]: This is going to read "a expert."

**Review by bill-transue — CHANGES_REQUESTED** [review]
Good work @ZASMan, although I think we may need to move the entire note into a translation because right now it will say "a expert" instead of "an expert".

**Comment by ZASMan** [comment]
Grammar will cost extra

File: app/views/ui-components/v1/forms/broker_registration/_broker_agency_information.html.slim (hunk 1 above)
Comment (ZASMan) [reply app/views/ui-components/v1/forms/broker_registration/_broker_agency_information.html.slim]: I'll change the whole thing to a translation

File: db/seedfiles/translations/en/me/insured.rb [hunk 2]
```diff
@@ -189,7 +189,8 @@
   :'en.registered?' => "Registered?",
   :'en.market' => "Market",
   :'en.issuer' => "Issuer",
-  :'en.broker' => "Expert",
+  :'en.broker' => "Broker",
+  :'en.broker_agency_note' => "",
```
Comment (bill-transue) [inline db/seedfiles/translations/en/me/insured.rb:193]: Looks like you forgot the `broker_agency_note` for ME.

**Review by bill-transue — CHANGES_REQUESTED** [review]
I think you're missing the broker agency note for ME.

File: db/seedfiles/translations/en/me/insured.rb (hunk 2 above)
Comment (bill-transue) [reply db/seedfiles/translations/en/me/insured.rb:193]: Apparently this is what they want. ¯\_(ツ)_/¯

**Pull request merged**